sycl/test-e2e/Basic/memory-consumption.cpp

```diff
@@ -7,6 +7,8 @@
 // extra memory allocation on device creation.
 // Issue #661: The test is failing sporadically on HIP AMD.
 // UNSUPPORTED: windows, opencl, hip_amd
+// Issue #9008: The test is failing sporadically on all hardware
+// UNSUPPORTED: *
```

Comment:
```suggestion
// REQUIRES: TEMPORARY_DISABLED
```

Reply:
I actually wanted to close the issue yesterday.
Did you find this test is still failing yesterday or today?

Reply:
Today - https://github.com/intel/llvm/actions/runs/4689271741/jobs/8311985618